libs/service-portal/occupational-licenses/src/lib/navigation.ts

```diff
@@ -11,6 +11,10 @@ export const occupationalLicensesNavigation: PortalNavigationItem = {
   },
   description: m.occupationalLicensesNavIntro,
   children: [
+    {
+      name: om.myLicenses,
+      path: OccupationalLicensesPaths.OccupationalLicensesRoot,
```

Comment:
hide from breadcrumbs?

Reply:
![image](https://github.com/island-is/island.is/assets/2184845/e73540b5-2e71-4bf1-ad50-c358779a4312)

Reply:
ahhhh my bad. looks good 💯